global.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Do we need this for this PR?

Reply:
Hmm we _did_ need this when jumping to `net480` since the 3.1 SDK wasn't compatible with building the tests. But for `net461` I'm not certain. I can check when I split the PR up.

Reply:
Update: The tests built fine without this change on `net461`, so looks like no it's not needed.